index.js

```diff
@@ -478,23 +483,46 @@ function modifyFirstrun() {
     });
 }
 
+/**
+ * Overrides time interval defauls from config file.
+ */
+function overrideDefaults() {
+    try {
+        // load the config file
+        let config = self.data.load('./config.json');
+
+        // if the file existed, parse the contents to a JSON object
+        if (config) {
+            config = JSON.parse(config);
+            // override time intervals with values from config
+            defaultSidebarInterval = config.defaultSidebarInterval;
+            timeElapsedFormula = config.timeElapsedFormula;
+            waitInterval = config.waitInterval;
+        }
+    } catch(e) {
+        console.error(e);
```

Comment:
Per the comment section of the readme, should this still throw an error? Leaving out the config file may be intentional, and it might throw someone off to get an error for this reason. Maybe we should add a comment saying something like `console.error("Either no config.json file was created, or it was placed in the wrong location. Error for debugging: " + e);`